### PR TITLE
refactor:added new title-added favicon-fixed horizontal scroll of hom…

### PR DESCRIPTION
**Pull request by tushargoyal812**
https://user-images.githubusercontent.com/69681896/169257486-30ef7b27-3096-4c9a-b7a2-783a498a4c5a.mp4

- ### Added new title
- ### Added favicon
- ### Fixed horizontal scroll of home page
- ### Changed hero image
- ### Aligned filters to the left
- ### Fixed cart count bug
- ### Fixed on logout items should be removed.

### CODE REVIEW

File: src/pages/cart/cart.js [hunk 1]
```diff
@@ -8,12 +8,14 @@ import { addToCartHandler } from "../../util-functions/add-to-cart"
 import axios from "axios"
 import {useNavigate} from 'react-router-dom'
 import { qunatityHandler } from "../../util-functions/qty-handler"
+import { useState } from "react"
 
 
 export const Cart=()=>{
     const {cart,setCart,setCartCount}=useCart()
     const {setWishlist,setWishlistCount,wishlist}=useWishlist()
     console.log(cart,"from cart");
```
Comment (deepak29-git) [inline src/pages/cart/cart.js]: remove console

**Review by deepak29-git — APPROVED** [review]
well folder structure and code look clean

File: src/components/banner/banner.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import bannerImage from '../../assets/E-commerce-banner.jpg'
```
Comment (deekshithmd) [inline src/components/banner/banner.js:1]: if possible try to use hosted images

**Review by deekshithmd — APPROVED** [review]

File: src/pages/cart/cart.js (hunk 1 above)
Comment (tushargoyal812) [reply src/pages/cart/cart.js]: done

File: src/components/banner/banner.js (hunk 2 above)
Comment (tushargoyal812) [reply src/components/banner/banner.js:1]: ok

**Pull request merged**